src/common/types/Flat.ts

```diff
@@ -1,38 +1,50 @@
-import generateID from "../helpers/generateID";
-
 export interface Flat {
-    id: string,
+    id?: number,
     name: string,
     address: FlatAddress,
     rooms?: number,
     area?: number,
-    facilities: string[],
-    images: string[],
+    numberOfGuests?: number,
+    active: boolean,
+    facilities: Facility[],
+    images: Image[],
     description: string
 }
 
```

Comment:
ok wiem, że to nie najlepszy moment, ale czy my jeszcze dodajemy od kiedy do kiedy jest mieszkanie zajęte?

Reply:
To nie jest od kiedy do kiedy zajete tylko od kiedy do kiedy wolne i to jest w bazie, wszystkiego nie łączyłem z backendem tylko żeby jakkolwiek działało bez błędów